src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -29,7 +31,12 @@ const modules = {
 		name: {
 			value: "GitHub"
 		},
-		url: "https://github.com"
+		url: "https://github.com",
+		actions: {
+			createBranch: {
+				templateUrl: `${envVars.APP_URL}/github/create-branch?issue_key={issue.key}&issue_summary={issue.summary}`
```

Comment:
Just something to avoid conflict later on. 
There already is a PR that removes the usage of all the `envVars` -> [link](https://github.com/atlassian/github-for-jira/pull/1557). So might suggest to just use `process.env.APP_URL`

Reply:
nah, leave it as is until mine is merged.  Either mine is merged first and he has to change his, or his is merged first and I need to fix it in mine.